.gitattributes

```diff
@@ -0,0 +1 @@
+dangerzone/gvisor_wrapper/entrypoint.py    eol=lf
```

Comment:
I was thinking more in the lines of `* text=auto`, which is arguably the most sensible choice and won't bite us for other files as well. Are you ok with that?

Reply:
Doing it for every file might be okay.

I just wonder how that would play with `.jpg`, `.gif` and `.png`. In case we don't want to change these ones we could add

```
*.jpg           -text
*.gif           -text
*.png           -text
```

Reply:
Hm, I thought that Git could distinguish between text and binary files. You're right though, I've seen `.gitattributes` doing what you propose, and it's probably better than what we have now.

Reply:
I updated the PR accordingly in [`cb87c45` (#838)](https://github.com/freedomofpress/dangerzone/pull/838/commits/cb87c456bcb8f14e00a506f99a761883a70e22a5)